sycl/test/scheduler/ReleaseResourcesTest.cpp

```diff
@@ -20,12 +20,10 @@ int main() {
 
   // Checks creating of the second host accessor while first one is alive.
   try {
-    sycl::default_selector device_selector;
-
     sycl::range<1> BufSize{1};
     sycl::buffer<int, 1> Buf(BufSize);
 
-    TestQueue Queue(device_selector);
+    TestQueue Queue(sycl::default_selector{});
```

Comment:
Hm... I didn't catch: what has changed here?
Didn't you intend to use `sycl::default_selector_v`?